lua/rustaceanvim/config/init.lua

```diff
@@ -58,17 +58,11 @@ vim.g.rustaceanvim = vim.g.rustaceanvim
 ---@field executor? RustaceanExecutor | executor_alias
 ---@field on_initialized? fun(health:RustAnalyzerInitializedStatus) Function that is invoked when the LSP server has finished initializing
 ---@field reload_workspace_from_cargo_toml? boolean Automatically call `RustReloadWorkspace` when writing to a Cargo.toml file
----@field hover_actions? RustaceanHoverActionsOpts Options for hover actions
+---@field replace_builtin_hover ? boolean whether use plugin's hover to replace Neovim's built-in `vim.lsp.buf.hover`.
+---@field float_win_config table Options applied to floating windows. See `vim.lsp.util.open_floating_preview`.
```

Comment:
issue:

- The field should be optional (`?` after the field name).
- Link to `api-win_config` using vimdoc syntax (vimdoc is generated from these lua docs).

```suggestion
---@field float_win_config? table Options applied to floating windows. See |api-win_config|.
```